packages/picasso/src/Section/styles.ts

```diff
@@ -64,4 +64,12 @@ export default ({ sizes, palette }: Theme) =>
       borderBottom: 'none',
       borderRadius: sizes.borderRadius.medium,
```

Comment:
I think we can make the animation for the section with `headerBar` look even better if we play with timings a little bit. There is weird interaction when we are closing the section right now with borderRadius and border.

```suggestion
      // we can make transition on borderWidth
      borderBottomWidth: '0px',
      borderRadius: sizes.borderRadius.medium,
      // default timing for collapse component
      transition-delay: '300ms',
```